src/openloco/objects/industry_object.h

```diff
@@ -8,9 +8,15 @@ namespace openloco
     struct industry_object
     {
         string_id name;
-        uint8_t pad_02[0xDE - 0x02];
-        uint8_t var_DE[2];
-        uint8_t var_E0[3];
+        uint8_t pad_02[0xCA - 0x02];
+        uint16_t var_CA; // start year?
+        uint16_t var_CC; // end year?
+        uint8_t var_CE;
+        uint8_t cost_fact; // 0xCF
+        uint8_t cost_ind;  // 0xD0
```

Comment:
I'm not sure I understand what these variables mean. Perhaps the following are more intuitive? (Assuming I understood their meaning correctly.)
```suggestion
        uint8_t cost_currency; // 0xCF
        uint8_t cost_index;    // 0xD0
```

Reply:
They are used for calculating industry build cost in L633:
`industryCost = industryObj->cost_fact * currencyMultiplicationFactor[industryObj->cost_ind] / 8;`
would 'cost_factor' be better than 'cost_currency'?

Reply:
We have used `ind` and `fact` for vehicle objects. I think this naming is fine as it is at least consistent.

Reply:
Ah, `factor` and `index` are much clearer, I think. Let's not save on bytes for this sort of thing.